common/src/main/java/com/wynntils/core/chat/tab/ChatTab.java

```diff
@@ -2,7 +2,7 @@
  * Copyright © Wynntils 2022.
  * This file is released under AGPLv3. See LICENSE for full license details.
  */
-package com.wynntils.core.chat;
+package com.wynntils.core.chat.tab;
```

Comment:
I think these can move back, for now. We have unfinished business with chat, but lets save that for another day.